djautotask/sync.py

```diff
@@ -320,11 +311,26 @@ def _assign_field_data(self, instance, object_data):
         self.set_relations(instance, object_data)
         return instance
 
-    def get_related_instance(self, relation_id):
-        return models.TicketStatus.objects.get(value=relation_id)
+    def get_related_instance(self, relation_id, object_field):
```

Comment:
I might be blind but I don't see anything calling this method

Reply:
Nope, that should have been removed. Fixed now.